### PR TITLE
[Admin] feature specs fixes

### PR DESCRIPTION
**Pull request by elia**
## Summary

These changes will help make feature specs work and properly load JS when driving a selenium browser.

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [ ] I have written a thorough PR description.
- [ ] I have kept my commits small and atomic.
- [ ] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

The following are not always needed:

- 📖 I have updated the README to account for my changes.
- 📑 I have documented new code [with YARD](https://www.rubydoc.info/gems/yard/file/docs/Tags.md).
- 🛣️ I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- ✅ I have added automated tests to cover my changes.
- 📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

File: admin/app/views/layouts/solidus_admin/application.html.erb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="<%= current_locale %>">
   <head>
+    <%= favicon_link_tag 'favicon.ico' %>
```
Comment (kennyadsl) [inline admin/app/views/layouts/solidus_admin/application.html.erb:4]: What is the error here, just for the sake of curiosity? Something related to a11y?

**Review by kennyadsl — APPROVED** [review]
👍

File: admin/app/views/layouts/solidus_admin/application.html.erb (hunk 1 above)
Comment (elia) [reply admin/app/views/layouts/solidus_admin/application.html.erb:4]: No, not accessibility, it's just capybara complaining for the 500 received when trying to fetch the default `/favicon.ico`.

**Review by rainerdema — APPROVED** [review]
👍

**Pull request merged**